devices.go

```diff
@@ -13,6 +13,7 @@ import (
 	"net/http"
 
 	gcpiotcore "cloud.google.com/go/iot/apiv1"
+	cbiotcore "github.com/clearblade/go-iot"
 	"golang.org/x/exp/maps"
 	"google.golang.org/api/iterator"
 	gcpiotpb "google.golang.org/genproto/googleapis/cloud/iot/v1"
```

Comment:
Since the cloud.google.com/go/iot/apiv1 version has been updated from v1.2.0 -> v1.4.0, we can use types from "cloud.google.com/go/iot/apiv1/iotpb". This older package has been deprecated.